github/resource_github_repository_test.go

```diff
@@ -68,6 +68,69 @@ func TestAccGithubRepositories(t *testing.T) {
 
 	})
 
+
+	t.Run("updates a repositories name without error", func(t *testing.T) {
+
+		oldName := fmt.Sprintf(`tf-acc-test-rename-%[1]s`, randomID)
+		newName := fmt.Sprintf(`%[1]s-renamed`, oldName)
+
+		config := fmt.Sprintf(`
+			resource "github_repository" "test" {
+			  name         = "%[1]s"
+			  description  = "Terraform acceptance tests %[2]s"
+			}
+		`, oldName, randomID)
+
+		checks := map[string]resource.TestCheckFunc{
+			"before": resource.ComposeTestCheckFunc(
+				resource.TestCheckResourceAttr(
+					"github_repository.test", "name",
+					oldName,
+				),
+			),
+			"after": resource.ComposeTestCheckFunc(
+				resource.TestCheckResourceAttr(
+					"github_repository.test", "name",
+					newName,
+				),
+			),
+		}
+
+		testCase := func(t *testing.T, mode string) {
+			resource.Test(t, resource.TestCase{
```

Comment:
Is there a way to add a check in here that the existing resource wasn't destroyed/a new resource wasn't created?

Reply:
We manually verified that the rename operation did not cause a repository deletion by viewing audit logs for our test org:

<img width="590" alt="CleanShot 2021-04-09 at 10 32 45@2x" src="https://user-images.githubusercontent.com/5034582/114196132-175def00-991f-11eb-99c5-7ec9b92c9aad.png">